doc/source/tutorial-slam-for-beginners-the-basics.rst

```diff
@@ -22,13 +22,14 @@ recommend the following online resources:
     </div> <br/> <hr><br/>
 
 
-- These `taped seminars <https://www.youtube.com/watch?v=U6vr3iNrwRA&list=PLgnQpQtFTOGQrZ4O5QzbIHgl3b1JHimN_>`_ by Cyrill Stachniss (2012-2013):
+- These `taped seminars <https://www.youtube.com/watch?v=U6vr3iNrwRA&list=PLgnQpQtFTOGQrZ4O5QzbIHgl3b1JHimN_>`_ by Cyrill Stachniss (2013-2014):
 
 .. raw:: html
 
     <div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-        <iframe src="https://www.youtube.com/embed/V9qQc5X7O0k" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
+        <iframe src="https://www.youtube.com/embed/U6vr3iNrwRA?list=PLgnQpQtFTOGQrZ4O5QzbIHgl3b1JHimN_"" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
```

Comment:
Thanks for the PR, good point!

However, I think you added double quotes at the end here, right? Also, the "embed" interface **seems** not to support the `?list=xxx` part, could you please remove it? Thanks.

Reply:
Woops, good catch! 

I was able to get the iframe to render properly with the list param on plain HTML, but I didn't get to test with this repository's markup generation, so I've removed that param from the src.